CPC-CHARTER.md

```diff
@@ -109,6 +109,10 @@ one-fourth of the Voting CPC membership to be affiliated with the same employer.
 Such a situation must be immediately remedied by the resignation or
 removal of one or more voting CPC members.
 
+If a Voting CPC member connected to an impact or non impact project is being
+replaced by someone from the same project, a sign off from the current active
+CPC member is required with an approval.
+
```

Comment:
I'm thinking maybe something like:

If a voting CPC member steps down before the end of their one year term they may be replaced using the same process by which they were elected. In this case the member will serve out the remainder of the term and must be re-confirmed at the end of the original one year term as if they had served for the full year.  

I think this is a bit more general, allows for a replacement and captures how long they will server for.

Reply:
In terms of needing approval from the original member. I think once they have officially stepped down it's up to the project and we don't necessarily need their approval.

Reply:
Edited my suggestion to also cover the case where they are removed by the CPC. This would hand the case where they can no longer be contacted.

Reply:
@mhdawson I think your suggestion is great. I had already approved, I guess I could have withdrawn but I wouldn't have come up with good wording!

Reply:
Updated wording.